bill-of-materials.json

```diff
@@ -472,7 +472,7 @@
 		"licenses": [
 			{
 				"type": "Apache License 2.0",
-				"confidence": 1
+				"confidence": 0.9988925802879292
```

Comment:
question: is this automatically updated by the script?

Reply:
Yes, verify failed. I assume it calculates distance to license with those values unfilled.